.github/workflows/sync-gitee.yml

```
@@ -19,6 +19,11 @@ jobs:
     
     steps:
     
+    - name: Harden the runner (Audit all outbound calls)
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)